pram

```diff
@@ -232,6 +261,8 @@ main() {
 			;;
 		*)
 			# a local file maybe?
+			[[ ! ${partof} || -n ${link_to} ]] || 
+				die "specifying --link-to is mandatory with local files when using --part-of"
```

Comment:
Let's do that only if `-p` was passed explicitly, and just disable `partof` if it was `def`.

Reply:
Done, in a fixup commit this time :)